source/stylesheets/_refills-header.scss

```diff
@@ -88,14 +88,25 @@
     @include refills-inset;
     border-radius: 30px;
     color: darken(desaturate($refills-logo-color, 50%), 15%);
-    display: inline;
+    display: inline-block;
     font-family: $refills-narrow-sans;
     font-size: 0.8em;
     font-weight: 400;
+    margin-bottom: 4em;
     padding: 0.5em 1.4em;
 
     @include media($refills-large-screen) {
       font-size: 0.8em;
     }
   }
+
+  .empties-banner {
+    background: #FCFFE0;
+    border-radius: 2em;
+    box-shadow: 0 2px 0px #E3E8AA;
```

Comment:
Color literals like `#E3E8AA` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#E3E8AA` should be written as `#e3e8aa`<br>`0px` should be written without units as `0`